### PR TITLE
Added Twilio API for Form

### PR DESCRIPTION
**Pull request by dougblackjr**
Connects to Twilio API. On form submit, will send welcome text message to person completing form. Initial work, just a start for what we need.

### CODE REVIEW

**Review by mattstauffer — DISMISSED** [review]
@dougblackjr Looks like a solid start--do you know how to do tests in Express? Feels like we should shoot for as much test coverage up front as possible so the desired functionality is clear to new devs who join the project.

**Comment by dougblackjr** [comment]
@mattstauffer, I don't know much about Node, just started learning it this week tbh. I updated the SMS to match more of what we're trying to do (url/sms, text "woke" to ###, receive response). It's basic, but it'll get us started. Let me know what's next.

File: public/index.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
             <section class="intake">
                 <p>Congue aciliquis atis, faccummy aliqui ing vero commodolor dolesecte, nec eugiamconse conubia.</p>
 
-                <form onsubmit="return false">
+                <form onsubmit="return false;">
```
Comment (pascalpp) [inline public/index.html]: This no-op change is causing a conflict with other changes that have already merged out. Can you remove this semi-colon?

File: index.js [hunk 2]
```diff
@@ -29,13 +29,18 @@ app.use(expressValidator());
...
 
-// // Controllers
-// const phoneController = require('./controllers/phone');
-
-// app.post('/new_phone_call', phoneController.newCall);
-// app.get('/new_phone_call', phoneController.newCallTestGet);
-// app.post('/redir_call_for_zip', phoneController.redirectCall);
-// app.get('/redir_call_for_zip', phoneController.redirectCallTest);
+// ngrok For Testing
+var ngrok = require('ngrok');
```
Comment (pascalpp) [inline index.js]: we're using es6 convention. so let's use `const` instead of `var` here.

File: index.js [hunk 3]
```diff
@@ -29,13 +29,18 @@ app.use(expressValidator());
...
+ngrok.authtoken(process.env.NGROK_AUTHTOKEN, function(err, token) {});
+ngrok.connect(function (err, url) {});
+
+// Controllers
+var smsController = require('./controllers/sms.controller');
```
Comment (pascalpp) [inline index.js]: `const`

File: index.js [hunk 4]
```diff
@@ -29,13 +29,18 @@ app.use(expressValidator());
...
+
+// SMS Routes
+app.route('/sms')
+  .get(smsController.index)
+  .post(smsController.receiveWoke);
```
Comment (pascalpp) [inline index.js]: we're not using semi-colons in this repo. (i need to setup a lint:fix cmd so it's easy to keep this consistent.) Can you remove semicolons here and merge the upstream development branch and resolve and conflicts? Let me know if you have any questions about the upstream changes.

**Review by pascalpp — CHANGES_REQUESTED** [review]
Some code style changes and some conflicts to resolve.

File: public/index.html (hunk 1 above)
Comment (dougblackjr) [reply public/index.html]: Done on next commit.

File: index.js (hunk 4 above)
Comment (dougblackjr) [reply index.js]: Fixed in next commit

File: index.js (hunk 3 above)
Comment (dougblackjr) [reply index.js]: Fixed in next commit

**Review by dougblackjr — COMMENTED** [review]
Errors fixed on newest commit

**Review by dougblackjr — COMMENTED** [review]
I'm not terribly familiar with testing, as I'm new to node.js. I may not be of help here at this point.

**Comment by dougblackjr** [comment]
I made changes in the newest commits. Let me know your thoughts, @mattstauffer @pascalpp

File: index.js [hunk 5]
```diff
@@ -13,11 +7,15 @@ const methodOverride = require('method-override')
 const bodyParser = require('body-parser')
 const expressValidator = require('express-validator')
 const saveUser = require('./src/user/save_user')
+const dotenv = require('dotenv')
```
Comment (pascalpp) [inline index.js]: dotenv needs to be initialized before other modules, notably `debug`, which is why i put it first in the file. can you move this require back to the top?

File: index.js [hunk 6]
```diff
@@ -13,11 +7,15 @@ const methodOverride = require('method-override')
...
 const debug = require('debug')('app')
 
+// Load environment variables from .env file
+dotenv.load()
```
Comment (pascalpp) [inline index.js]: and these two lines back up top as well.

**Review by pascalpp — CHANGES_REQUESTED** [review]
this is looking good. I still need to try it out. Couple of minor changes.

**Review by dougblackjr — COMMENTED** [review]
Fixed on newest commit

**Review by pascalpp — APPROVED** [review]

**Pull request merged**